.gitignore

```diff
@@ -9,4 +9,10 @@ Cargo.lock
 
 # Editor configurations
 .idea/
-.vscode/
\ No newline at end of file
+.vscode/
+
+# Downloaded artifacts
+VapourSynth**.7z
```

Comment:
Vapoursynth installation happens in project folder for convenience by continuous integration but doesn't need to be that way for the usual workflow.
Location of _VapourSynth**.7z_ isn't required to be here.